samples/Basic_NFC/app/MFRC522.cpp

```diff
@@ -0,0 +1,1768 @@
+/*
+* MFRC522.cpp - Library to use ARDUINO RFID MODULE KIT 13.56 MHZ WITH TAGS SPI W AND R BY COOQROBOT.
+* NOTE: Please also check the comments in MFRC522.h - they provide useful hints and background information.
+* Released into the public domain.
+*/
+#include <user_config.h>
+#include <SmingCore/SmingCore.h>
+#include <AppSettings.h>
```

Comment:
Removed
Thanks for your help
Let me know if need more adjustments
I build and run the code and it still works
I will commit it now